book/website/community-handbook/consistency.md

```diff
@@ -0,0 +1,49 @@
+(ch-consistency)=
+# Achieving Retrospective Consistency
```

Comment:
Hi @paulowoicho!! As we discussed in person, could we change this landing page for the chapter to be a little more "timeless". Something like "Maintaining consistency" perhaps? The somewhere in the chapter you can acknowledge the work you're doing (and how you're doing it - writing guidelines, writing issues etc) rather than making the _retrospective_ aspect the headline of the chapter?